multigeiger/multigeiger.ino

```diff
@@ -488,16 +497,6 @@ void loop()
   // Check if there are enough pulses detected or if enough time has elapsed.
   // If yes, then it is time to charge the HV capacitor and calculate the pulse rate.
   if ((GMC_counts >= MAXCOUNTS) || ((current_ms - time2display) >= DISPLAYREFRESH )) {
-    // copy values from ISR
-    portENTER_CRITICAL(&mux_GMC_count);                               // enter critical section
-    GMC_counts = isr_GMC_counts;
```

Comment:
this can't be done in this if-block as the if-condition already expects GMC_counts to be set and "fresh".

Reply:
OK, i see the problem. Shall i merge or will you do it?

Reply:
merged :)